### DISCUSSION TITLE
3d motion canvas?

### DISCUSSION THREAD

**Discussion by wingsandstache** [motion-canvas/motion-canvas#1011] [Ideas]
Are there any plans to have a 3d motion canvas. I would love to animate with motion canvas but unfortunately I need to display 3d information.

**Comment by ClemDLS** [comment]
That would be awesome yes...

**Comment by aarthificial** [comment]
There are no specific plans right now but you can use ThreeJS to render 3D stuff.
Here's an example from one of my videos where I do it.
The `Three` component is used to display a ThreeJS scene:
https://github.com/motion-canvas/examples/blob/master/examples/deferred-lighting/src/components/Three.ts
And here's a sample usage:
https://github.com/motion-canvas/examples/blob/master/examples/deferred-lighting/src/scenes/normals.tsx#L42-L52
